icepyx/core/APIformatting.py

```diff
@@ -341,8 +325,17 @@ def _get_possible_keys(self) -> dict[str, list[str]]:
                     "Coverage",
                 ],
             }
+        else:
+            raise RuntimeError("Programmer error!")
```

Comment:
The typechecker doesn't seem to be good at knowing whether a conditional is "exhaustive", so we're making that explicit here.

Reply:
It would be good to have something more explicit (e.g. "exhaustive list error", since that seems to be a typechecker weakness) that makes it more obvious why it go there to begin with.

Reply:
Love the idea to go really specific with the name. :100:

Reply:
ea4a0c5